docs/UserGuide.md

```diff
@@ -142,6 +142,19 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Recording unpaid tuition fee of a student: `owe`
+
+Updates the amount of tuition fee owed by a specified student after a lesson.
+
+Format: `owe INDEX hr/HOURS_OWED`
+
+Example: 
+* `owe 2 hr/1.5` updates the tuition fee owed by the 2nd student in the list.
```

Comment:
Could perhaps include image of result after the example

Reply:
Thanks for the suggestion! I added just now!